DataFormats/HcalRecHit/src/classes_def.xml

```diff
@@ -118,4 +118,21 @@
   <class name="edm::reftobase::Holder<CaloRecHit,edm::Ref<edm::SortedCollection<HFRecHit,edm::StrictWeakOrdering<HFRecHit> >,HFRecHit,edm::refhelper::FindUsingAdvance<edm::SortedCollection<HFRecHit,edm::StrictWeakOrdering<HFRecHit> >,HFRecHit> > >" />
   <class name="edm::reftobase::Holder<CaloRecHit,edm::Ref<edm::SortedCollection<HORecHit,edm::StrictWeakOrdering<HORecHit> >,HORecHit,edm::refhelper::FindUsingAdvance<edm::SortedCollection<HORecHit,edm::StrictWeakOrdering<HORecHit> >,HORecHit> > >" />
   <class name="edm::reftobase::Holder<CaloRecHit,edm::Ref<edm::SortedCollection<HBHERecHit,edm::StrictWeakOrdering<HBHERecHit> >,HBHERecHit,edm::refhelper::FindUsingAdvance<edm::SortedCollection<HBHERecHit,edm::StrictWeakOrdering<HBHERecHit> >,HBHERecHit> > >" />
-</lcgdict>
+
+  <class name="hcal::HcalRecHitSoA"/>
+  <class name="hcal::HcalRecHitSoA::View"/>
+
+  <class name="hcal::RecHitHostCollection"/>
+  <read
+    sourceClass="hcal::RecHitHostCollection"
+    targetClass="hcal::RecHitHostCollection"
+    version="[1-]"
+    source="hcal::HcalRecHitSoA layout_;"
+    target="buffer_"
+    embed="false">
+  <![CDATA[
+    hcal::RecHitHostCollection::ROOTReadStreamer(newObj, onfile.layout_);
+  ]]>
+  </read>
```

Comment:
Can this be done via a `classess.cc` file instead, similar for example to `DataFormats/EcalRecHit/src/classes.cc` ?

Reply:
Done in 4615e8825352834ac1dd4f1bbe766b0cafc1d321